chandra_aca/maude_decom.py

```diff
@@ -1137,9 +1137,8 @@ def get_aca_packets(
     if not blobs and not frames:
         frames = True
 
-    # check that only one of blobs or frames is set
-    if blobs and frames:
-        raise ValueError("Specify only one of 'blobs' or 'frames'")
+    if (not blobs and not frames) or (frames and blobs):
+        raise ValueError("Specify one and only one of 'blobs' or 'frames'")
```

Comment:
Repeating the comment, basically that the version in 9d0a53 was actually correct.

OK, the original logic was unnecessarily complex. It was
```
assert frames XOR blobs  # one and only one of frame/blobs is set
```
However, the statement before ensures that at least one of them is always set, so then here (indeed) you can just check `blobs and frames`.

Maybe it would be worth a documentation update to reflect that if neither is set, then `frames` is the default.

Reply:
I made the other comment on code that got changed, so it might not show up.

Reply:
I see everything!